docs/manual-configuration.md

```diff
@@ -1,31 +1,30 @@
-# Manually configure the Elastic Distribution for OpenTelemetry Collector
-Collecting logs and host metrics with the Elastic Distribution for OpenTelemetry Collector without using the [guided onboarding](guided-onboarding.md) requires some manual configuration.
+# Manually configure the Elastic Distribution of OpenTelemetry Collector
+Collecting logs and host metrics with the Elastic Distribution of OpenTelemetry (EDOT) Collector without using the [guided onboarding](guided-onboarding.md) requires some manual configuration.
 
 ## Before you begin
-The Elastic OTel collector has the following requirements and limitations:
+The EDOT Collector has the following requirements:
 
 - The **Admin** role or higher is required to onboard system logs and metrics. To learn more, refer to [Assign user roles and privileges](https://www.elastic.co/docs/current/serverless/general/assign-user-roles).
 - Root privileges on the host are required to run the OpenTelemetry collector used in this quickstart.
-- The Elastic OTel Collector only works on Kubernetes, Linux, and MacOS systems.
-- Refer to [Elastic OpenTelemetry Collector limitations](collector-limitations.md) for known limitations when using the Elastic Distribution for the OpenTelemetry collector.
+- The EDOT Collector only works on Kubernetes, Linux, and MacOS systems.
 
 ## Collect your logs and metrics
 
-To manually configure the Elastic OTel Collector, gather the following information:
+To manually configure the EDOT Collector, gather the following information:
 
 - **Your Elasticsearch endpoint**: From the help menu in Kibana or your serverless Observability project, select **Connection details** and copy the **Elasticsearch endpoint**.
 - **API key**:
    - **Kibana:** From the help menu, select **Connection details** and select **Create and manage API keys**. From the **API keys** page, select **Create API key**. Give your API key a name, select **Create API key**, and copy the new API key.
    - **Serverless:** From the help menu, select **Connection details** and select the **API key** tab. Give your API key a name, select **Create API key**, and copy the new API key.
 
-Then manually configure the Elastic OTel Collector to collect logs and metrics on a MacOS or Linux system:
+Then manually configure the EDOT Collector to collect logs and metrics on a MacOS or Linux system:
 
 1. Download and extract the standalone Elastic Agent for your platform. For more on downloading and extracting a standalone Elastic Agent, refer to the first step in [Install standalone Elastic Agents](https://www.elastic.co/guide/en/fleet/current/install-standalone-elastic-agent.html).

```

Comment:
I see we are mixing the use of EDOT collector and standalone Elastic Agent, this is a tricky one so I don't have a specific solution but just a heads up that we should come up with a good definition of when we call it EDOT Collector and when we call it Elastic Agent.

Maybe we explain this upfront and then only use invoke EDOT Collector?

Reply:
I think it makes sense to explain that the EDOT collector uses the Elastic Agent as a wrapper, or however we want to phrase this. I'm not sure we can only use EDOT collector here, as the user needs to go into the agent directory to modify the otel config, and the directory is named `elastic-agent-*`.